jobs/refresh-zl-warehouse.yml

```diff
@@ -1,13 +1,34 @@
-type: "job-pipeline"
-description: "Refreshing data from all sites"
+type: "iterating-job"
+description: "Refreshing OpenMRS Data for all sites"
 schedule:
   cron: "${executeCron.refreshZlWarehouse}"
 configuration:
+  maxConcurrentJobs: 5  # Import into up to 5 sites concurrently
   errorHandling:
     maxAttempts: 3
     retryInterval: 60
     retryIntervalUnit: "MINUTES"
-  jobs:
-    - path: "load-from-openmrs-all-sites.yml"
-    - path: "load-warehouse-reference-tables.yml"
-    - path: "load-warehouse-analysis-tables.yml"
```

Comment:
Because we are removing the load-warehouse-xxx jobs from here, and putting the directly in the load-hiv-data job, we can remove the independent load-from-openmrs-all-sites job and just move it's contents into here, thus simplifying things.